web/html-forms.md

```diff
@@ -96,4 +125,29 @@ Following is an example which shows the code snippet to declare the Button Eleme
     <button type="button">Click Me!</button>
 </form>
 ```
+<form>
+    <button type="button">Click Me!</button>
+</form>
+
+#### Validation Attribute
+HTML elements which allows restriction in the form elements are the validation attribute.
+1. Readonly Attribute : This attribute do not allow any text to be entered in the input field.
+```
+<form>
+  <label for="name">Name:</label><br>
+  <input type="text" id="name" name="name" value="Shreya" readonly><br>
+  <label for="email">Email:</label><br>
+  <input type="email" id="email" name="email" value="shreyakapoor98@gmail.com">
+</form>
+```
+<form>
+  <label for="name">Name:</label><br>
+  <input type="text" id="name" name="name" value="Shreya" readonly><br>
+  <label for="email">Email:</label><br>
+  <input type="email" id="email" name="email" value="shreyakapoor98@gmail.com">
+</form>
```

Comment:
Why is same code written twice ?

Reply:
@Kundan28 I have just used the same labels, but the purpose of each is different like in this, the validation attribute is read only which i have applied on the input field

Reply:
Ohhh you mean the same code above and below, because the above code shows the code snippet and the below code shows the output of it in md file

Reply:
I will just send you the screenshot

Reply:
I don't see any difference in both the code.
So could you highlight it and show it to me ?

Reply:
I checked it.

Reply:
No need to send screenshot.

Reply:
![image](https://user-images.githubusercontent.com/31164665/80826821-7d66d000-8c00-11ea-98e0-b04b7644a9fc.png)

right side shows the preview

Reply:
Thanks, it's fine. Have a look at other comments that I made.